includes/class-wsuwp-person-display.php

```diff
@@ -90,7 +90,7 @@ public function person_permalink( $url, $post ) {
 	 *
 	 * @return string Path to the template file.
 	 */
-	public function theme_has_template() {
+	public static function theme_has_template() {
 		return locate_template( 'wsu-people-templates/person.php' );
```

Comment:
I would leave "templates" out of this and go with `wsu-people/person.php`. Makes the directory name a bit smaller and should be just as clear.